frontend/model/contracts/members/removeMemberSideEffect.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+import sbp from '~/shared/sbp.js'
+
+export default async function removeMemberSideEffect (data) {
+  const rootState = sbp('state/vuex/state')
```

Comment:
NOTE: @taoeffect
I had to move this function out of contracts/group.js, otherwise, it would create a circular dependency when imported at proposals.js because that file also imports contracts/group.js.

Not sure if this is the best place... any suggestion?

Reply:
I think having this inside of `group.js` and calling it via `await sbp(...` as you tried should be fine. You just were calling the wrong selector before (see comment below about that)